Interop/Tensorflow/src/main/java/org/tribuo/interop/tensorflow/TrainTest.java

```diff
@@ -88,24 +108,54 @@ public String getOptionsDescription() {
         @Option(charName='v',longName="testing-file",usage="Path to the libsvm format testing file.")
         public Path testingPath;
 
-        @Option(charName='b',longName="batch-size",usage="Test time minibatch size.")
+        @Option(charName='i',longName="init-name",usage="Name of the initialisation operation.")
+        public String initName = Init.DEFAULT_NAME;
+        @Option(charName='l',longName="output-name",usage="Name of the output operation.")
+        public String outputName;
+        @Option(longName="optimizer-param-names",usage="Gradient optimizer param names, see org.tribuo.interop.tensorflow.GradientOptimiser.")
+        public List<String> gradientParamNames = DEFAULT_PARAM_NAMES;
+        @Option(longName="optimizer-param-values",usage="Gradient optimizer param values, see org.tribuo.interop.tensorflow.GradientOptimiser.")
+        public List<Float> gradientParamValues = DEFAULT_PARAM_VALUES;
```

Comment:
This is a bit gnarly. I thought there was a way to pass maps with the Options system?